_posts/2023-11-09-in-progress-wildfly-feature-on-openshift.adoc

```diff
@@ -0,0 +1,276 @@
+```
+After running the commands above, you should expect to see the following output:
+```
+Waited for 1.068205845s due to client-side throttling, not priority and fairness,
+request: GET:https://api.sandbox-m4.g2pi.p1.openshiftapps.com:6443/apis/triggers.tekton.dev/v1beta1?timeout=32s
+NAME: simple-webapp-oidc
+LAST DEPLOYED: <current date and time>
+NAMESPACE: <your namespace>
+STATUS: deployed
+REVISION: 1
+TEST SUITE: None
+NOTES:
+To follow the deployment of your application, run:
+
+$ oc get deployment simple-webapp-oidc -w
+```
+Click on the three dots beside the pod's name and click on `Edit Deployment`. Make sure that the image name has `:latest`
+added to it. If not, edit it to reflect the correct image name under `Image Name`. Under `Show advanced image options`,
+set the `Pull Secret` to be the one you configured in the last section and click `Save`.
+
+Your application is now deploying. Click on the `simple-webapp-oidc` pod and under the `resources` tab, you should see the status of your deployment.
+Alternatively, you can use the following commands:
+```
+$ oc get deployment simple-webapp-oidc -w
+```
+
+== Get the Application URL
+
+Just like RH-SSO, you can find the url for your webapp by clicking on `Open URL` which will redirect you to the WildFly landing page. Alternatively, you can use the OC client to do this using the following commands: 
+```
+SIMPLE_WEBAPP_OIDC_URL=https://$(oc get route simple-webapp-oidc --template='{{ .spec.host }}') &&
+echo "" &&
+echo "Application URL:              $SIMPLE_WEBAPP_OIDC_URL/simple-webapp-oidc" &&
+echo "Valid redirect URI:           $SIMPLE_WEBAPP_OIDC_URL/simple-webapp-oidc/*" &&
+echo ""
+```
+
+== Finish Configuring RH-SSO 
+Copy the link for your webapp and go back to the admin console for RH-SSO and add `<Simple-webapp-oidc-url>/simple-webapp-oidc/*`
+to the `Valid Redirect URIs` field. Notice that the link starts with `https`. We will be changing it to `http` since we
+have not performed any ssl configuration.
+
+== Access the Application
+Now, let’s try accessing our application using <Simple-webapp-oidc-url>/simple-webapp-oidc. 
+
+Click on "Access Secured Servlet".
+
+Now, you’ll be redirected to RH-SSO's login page. If you click on the url on the search bar, you will see the scope values specified in the `redirect-uri` field with the different scope values separated by a `+`. You will also notice that a new scope value: `openid`. This indicates that we are going to be using OpenID Connect to authenticate the user. 
+
+Log in with `Alice` and the password that you set when configuring RH-SSO.
+
+Next, you’ll be redirected back to our application, and you should see the "Secured Servlet" page. That means that we were able to successfully log in to our application using the RH-SSO OpenID provider!
+
+This page will display the Current Principal, and a list of claim values obtained using the scope values you configured. This is what it will look like: 
```

Comment:
I think the "Current Principal" can be lowercase here

Reply:
Fixed!